presto-postgresql/src/test/java/io/prestosql/plugin/postgresql/TestPostgreSqlIntegrationSmokeTest.java

```diff
@@ -164,6 +164,17 @@ public void testTableWithNoSupportedColumns()
         }
     }
 
+    @Test
+    public void testInsertWithNotSupportedColumn()
+            throws Exception
+    {
+        // decimal(50,0) is not supported. Value inserted into column 'z' shouldn't be shifted to the omitted column 'y'
```

Comment:
Verify that `y` is indeed not supported.
Maybe replace
```
assertQuery("SELECT x, z FROM test_insert_not_supported_column", "SELECT 123, 'test'");
```

with 
```
assertQuery("SELECT * FROM test_insert_not_supported_column", "SELECT 123, 'test'");
```

...
but having explicit check using `information_schema.columns` would be better